src/Owin.Security.OpenIdConnect.Server/Events/ValidationEndpointContext.cs

```diff
@@ -14,7 +14,7 @@
 namespace Owin.Security.OpenIdConnect.Server {
     /// <summary>
     /// An event raised before the authorization server handles
-    /// the request made to the token validation endpoint.
+    /// the request made to the token introspection endpoint.
```

Comment:
Ideally, we'll be able to mention that in the incoming documentation. If you think it's worth a mention in the code base, feel free to add it under `ValidationEndpoint`/`OnValidationEndpoint` in `IOpenIdConnectServerProvider` and `OpenIdConnectServerProvider` :+1: